modules/apps/commerce/commerce-shipping-engine-fixed-web/src/main/java/com/liferay/commerce/shipping/engine/fixed/web/internal/frontend/CommerceShippingFixedOptionClayTable.java

```diff
@@ -173,13 +173,22 @@ public int getItemsCount(
 			HttpServletRequest httpServletRequest, Filter filter)
 		throws PortalException {
 
+		ThemeDisplay themeDisplay =
```

Comment:
@alecsloan using ThemeDisplay in this context (JaxRS) is to consider an antipattern, or in general I try to avoid it when there are no "Theme" related things.  Instead use _portal.getCompanyId, or directly the commerceShippingMethod.getCompanyId

//cc @dsitu